test/collection_form_test.rb

```diff
@@ -106,6 +110,45 @@ def initialize(house)
     proc { @rooms.first.reload}.must_raise ActiveRecord::RecordNotFound
   end
 
+  describe "reject_if option as a symbol" do
```

Comment:
How about as a Proc / with different arity? IMO either we care about all those cases enough to add tests for all of them, or we don't care about them enough to have them at all (not sure how much we care about feature parity with ActiveRecord)